### PR TITLE
new graph method (inst_dist) & graphs

### PR DESCRIPTION
**Pull request by Carol217**
Added the method that'll graph the median distance between male pronoun instances and the median distance between female pronoun distances as ration of M:F and F:M depending on which is greater. A few graphs are attacked, though they will be modified based on later data provided by Shobhita

### CODE REVIEW

**Comment by Carol217** [comment]
I realize that for some reason I have commits in the .txt novel files, and I think that's just something my computer automatically does despite me not touching or accessing them through anything but the Corpus class. I can't quite undo so.

**Comment by ltagliaferri** [comment]
Yeah, we want to avoid changing the commits that affect other files. You may need to move your changes elsewhere and then reset your local Git repo before moving the changes back in and updating this pull request.

File: gender_novels/analysis/analysis.py [hunk 1]
```diff
@@ -285,6 +290,161 @@ def instance_dist(novel, word):
...
+    fig.tight_layout()
+    # plt.show()
+    filepng = "visualizations/" + title + ".png"
+    filepdf = "visualizations/" + title + ".pdf"
+    plt.savefig(filepng, bbox_inches='tight')
+    plt.savefig(filepdf, bbox_inches='tight')
+
+
+def run_dist_inst():
+    corpus = Corpus('sample_novels')
```
Comment (mscuthbert) [inline gender_novels/analysis/analysis.py]: this needs at least documentation, if not doctests.  Lab policy is: write documentation, then code.

Why is 10 hardcoded?  What does it mean?

File: gender_novels/analysis/analysis.py [hunk 2]
```diff
@@ -285,6 +290,161 @@ def instance_dist(novel, word):
     return output
 
 
+def process_medians(helst, shelst, authlst):
+    """
```
Comment (mscuthbert) [inline gender_novels/analysis/analysis.py]: this needs at documentation and doctests.  Lab policy is: write documentation, then code.

**Pull request closed without merging**